week-3/implement/is-proper-fraction.js

```diff
@@ -33,3 +33,41 @@
 // Explanation: The fraction 3/3 is not a proper fraction because the numerator is equal to the denominator. The function should return false.
 
 // These acceptance criteria cover a range of scenarios to ensure that the isProperFraction function handles both proper and improper fractions correctly and handles potential errors such as a zero denominator.
+
+///...............................................
+
+
+
+function isProperFraction(numerator, denominator) {
+    if (denominator === 0) {
+      return undefined;
+    }
+   if (numerator < denominator && numerator >= 0) {
```

Comment:
you could have `-1/7` this is a proper fraction, your code will say other wise at the moment.